dask/dataframe/core.py

```diff
@@ -3946,6 +3946,11 @@ def quantile(df, q):
     q : list/array of floats
         Iterable of numbers ranging from 0 to 100 for the desired quantiles
     """
+    # current implementation needs qs to be sorted, sort in-place to make sure
+    q = np.asanyarray(q)
+    if q.ndim > 0:
+        q.sort(kind='mergesort')
```

Comment:
Is it possible that this modifies the input provided by the user?  If so then we probably don't want to sort in place.

```python
In [1]: import numpy as np

In [2]: x = np.array([1, 2, 3])

In [3]: np.asanyarray(x) is x
Out[3]: True

In [4]: np.sort(x)
Out[4]: array([1, 2, 3])

In [5]: np.sort(x) is x
Out[5]: False
```

Reply:
Glad you noticed that. Of course it will sort the user's array if it was an ndarray to begin with. I'll fix it.